qa/suites/rgw/sts/tasks/webidentity.yaml

```diff
@@ -0,0 +1,15 @@
+tasks:
+- tox: [ client.0 ]
+- keycloak:
+    client.0:
+- s3tests:
+    client.0:
+      force-branch: master
```

Comment:
so should this one. but do it after the PR has been cleanly run through teuthology

Reply:
Okay.

Reply:
Actually you might want to leave this in and set `force-branch` to `ceph-master`. Take a look at what the normal s3-tests task is doing and do that same thing

Reply:
Fine, I was about to ask the same thing.